packages/react-components/src/atoms/__tests__/Button.test.tsx

```diff
@@ -125,12 +125,6 @@ it('renders a stretched small button', () => {
   expect(getComputedStyle(screen.getByRole('button')).flexGrow).toBe('1');
 });
 
-it('renders a non-stretched small button', () => {
-  render(<Button small stretch={false} />);
-
-  expect(getComputedStyle(screen.getByRole('button')).flexGrow).toBe('');
-});
-
```

Comment:
Not sure if I understand it correct but I guess we'll need tests for the new `noMargin` prop?

Reply:
This was a test for the noStrech prop. It was unnecessary because you can make it not stretch by changing the parent elements display to inline-block. There is a form of technical debt called prop creep and the Link and Button components are struggling with it so I am trying to reduce it where i can